tests/e2e/normalize.go

```diff
@@ -81,6 +81,11 @@ func normalizeKRMObject(u *unstructured.Unstructured, project testgcp.GCPProject
 	// Specific to VertexAI
 	visitor.replacePaths[".status.blobStoragePathPrefix"] = "cloud-ai-platform-00000000-1111-2222-3333-444444444444"
 
+	// Specific to BigTable
+	visitor.replacePaths[".metadata.finishTime"] = "2024-04-01T12:34:56.123456Z"
+	visitor.replacePaths[".metadata.requestTime"] = "2024-04-01T12:34:56.123456Z"
+	visitor.replacePaths[".instances[].createTime"] = "2024-04-01T12:34:56.123456Z"
+
```

Comment:
Though it has the same result today, I guess we want to put this in the `normalizeHTTPResponses`, since `normalizeKRMObject` from the name is more for the golden object.

Reply:
Great call - will do!